src/services/profile/viewUser.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response } from "express";
+import { AuthRepository } from "../../Repository/Auth";
+const auth = new AuthRepository();
+
+const viewCurrentUser = async (req:Request, _res:Response) => {
+    const getuser = auth.getUser(req.params.id)
```

Comment:
Change this name as recommended above

Reply:
changed auth to new AuthRepository () as it's constructor signature